src/rules/number-max-precision/index.js

```diff
@@ -0,0 +1,47 @@
+import { isNumber } from "lodash"
+import execall from "execall"
+import {
+  report,
+  ruleMessages,
+  validateOptions
+} from "../../utils"
+
+export const ruleName = "number-max-precision"
+
+export const messages = ruleMessages(ruleName, {
+  expected: (number, precision) => `Unexpected precision: ${number} should be ${number.toFixed(precision)}`,
```

Comment:
"expected" and "unexpected" are together here.

I think we should follow `color-hex-length/case`'s lead and simply have `Expected ${number} to be ${number.toFixed(precision)}`, which would produce something like:

'Expected "0.11111" to be 0.11" (number-max-precision)'